src/NamingStrategy/MapNamingStrategy.php

```diff
@@ -52,8 +52,21 @@ public static function createFromHydrationMap(array $hydrationMap) : self
     /**
      * @param array<string, string> $extractionMap
      * @param array<string, string> $hydrationMap
+     * @deprecated Use createFromAsymmetricMap
      */
-    public function createFromAssymetricMap(array $extractionMap, array $hydrationMap) : self
+    public static function createFromAssymetricMap(array $extractionMap, array $hydrationMap) : self
+    {
+        $strategy = new self();
```

Comment:
Please don't duplicate any code. Use method `createFromAsymmetricMap` here.

Reply:
@froschdesign shall I then straight remove `createFromAssymetricMap` (because it can never be called due to private ctor) ?

Reply:
Right, that makes more sense.